File: java/CPU.java
public class CPU {
  
        String name;
        int price;
    
    
        CPU(){
    
        }
    
        CPU(String n, int p){
            this.name = n;
            this.price = p;
        }
    
        void setCPUname(String n){
            this.name = n;
         
        }
    
        void setCPUPrice(int p){
            this.price = p;
        }
    
        String getCPUname(){
            return this.name;
        }
    
        int getCPUPrice(){
            return this.price;
        }
    
    
}

File: PHP/CPU.php
<?php

class CPU{
    private $name;
    private $price;
    
   

    public function __construct($name = 0, $price = 0) {
        $this->name = $name;
        $this->price = $price;

    }

    public function setCPUName($name) { //
        $this->name = $name;
    }
    public function getCPUName() { 
        return $this->name;
    }

    public function setCPUPrice($price) { //
        $this->price = $price;
    }

    public function getCPUPrice(){ 
        return $this->price;

    } 

}
?>